### PR TITLE
V1

### PR DESCRIPTION
**Pull request by estebanmino**
Working with WC v2 beta.17, waiting for updates.

WIP for WC v1

### CODE REVIEW

**Comment by estebanmino** [comment]
working with WC v1 now

File: example/dapps/helpers/userAgent.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+const userAgentIsMobile = (): boolean => {
```
Comment (brunobar79) [inline example/dapps/helpers/userAgent.ts:1]: @walletconnect/browser-utils exports a function `isMobile`, probably a good idea to use that instead

File: stories/Thing.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import React from 'react';
```
Comment (brunobar79) [inline stories/Thing.stories.tsx:1]: If we are not using stories at all, we should remove this folder and the storybook dependency

File: test/blah.test.tsx [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import * as ReactDOM from 'react-dom';
+import { Default as Thing } from '../stories/Thing.stories';
+
+describe('Thing', () => {
+  it('renders without crashing', () => {
```
Comment (brunobar79) [inline test/blah.test.tsx:6]: is it worth to tweak this test and render the button?

**Review by brunobar79 — APPROVED** [review]
Amazing 🥇 

On separate PRs we should take care of:

- Creating a README for devs
- Add a script to publish to npm
-  Fix styles on the example app (background color, fonts, etc)
- Publishing the demo to github pages

File: example/dapps/v1/hooks.ts [hunk 4]
```diff
@@ -0,0 +1,37 @@
+import WalletConnect from '@walletconnect/client';
+import { useDispatch, useSelector } from 'react-redux';
+import { createSelector } from 'reselect';
+import {
```
[the remark this answers is not in the document]
Comment (estebanmino) [reply example/dapps/v1/hooks.ts:4]: oh true

File: example/dapps/v2/hooks.ts [hunk 5]
```diff
@@ -0,0 +1,48 @@
+import WalletConnectClient from '@walletconnectv2/client';
```
[the remark this answers is not in the document]
Comment (estebanmino) [reply example/dapps/v2/hooks.ts:1]: v2 work in https://github.com/rainbow-me/rainbow-button/pull/2

**Pull request merged**